ansible/group_vars/all

```diff
@@ -62,6 +62,9 @@ runtimesManifest:
       default: true
       image:
         name: "swift3action"
+    - kind: "swift:3.1.1"
```

Comment:
@rabbah @paulcastro should we make the 3.1.1 the new default now, then we need to mode `default: true` down to 3.1.1